system/libarc32_arduino101/common/scss_registers.h

```diff
@@ -38,6 +38,8 @@
 #define MMIO_REG_ADDR(addr) ((volatile uint32_t *)(addr))
 #define MMIO_REG_VAL_FROM_BASE(base, offset) \
     (*((volatile uint32_t *)((base)+(offset))))
+    
+#define REG_VAL(addr) (*(uint32_t *)(addr))
```

Comment:
It was not necessary to use volatile for all my register accesses in the Power Management library. I could just make everything volatile which should also work.